sharding/sharding.go

```diff
@@ -213,7 +213,7 @@ func (r *ShardingFerry) copyPrimaryKeyTables() error {
 	r.config.TableFilter.(*ShardedTableFilter).PrimaryKeyTables = pkTables
 	r.config.CopyFilter.(*ShardedCopyFilter).PrimaryKeyTables = pkTables
 
-	sourceDbTables, err := ghostferry.LoadTables(r.Ferry.SourceDB, r.config.TableFilter, r.config.CompressedColumnsForVerification, r.config.IgnoredColumnsForVerification)
+	sourceDbTables, err := ghostferry.LoadTables(r.Ferry.SourceDB, r.config.TableFilter, r.config.CompressedColumnsForVerification, r.config.IgnoredColumnsForVerification, r.config.CascadingPaginationColumnConfig)
```

Comment:
Question: this is calling to the PrimaryKeyTables, which are tables where there's only one entry consists of the sharding key. Does this need this option passed in? Should we explicitly not pass it in? @hkdsun 

(This is not an objection, I just want to make sure we're doing the right thing as I'm less familiar with PrimaryKeyTables).